ilastik/shell/gui/reportIssueDialog.py

```diff
@@ -0,0 +1,120 @@
+
+        # Create and add the instruction label
+        instruction_label = QLabel()
+        instruction_label.setText(
+            "<p>Sorry you ran into a problem! Please include the information below when you report your issue.</p>"
+            f'<p>The best place to get help is the <a href="{FORUM_URI}">community forum</a>. '
+            "Most posts get responses very quickly there, and others can benefit from the advice you receive!</p>"
+            f'<p>Alternatively, you can email us at <a href="mailto:{TEAM_EMAIL}">{TEAM_EMAIL}</a>. '
+            "We'll do our best to get back to you quickly.</p>"
+        )
+        instruction_label.setOpenExternalLinks(True)
+        instruction_label.setWordWrap(True)
+        instruction_label.setMaximumHeight(150)
+        main_layout.addWidget(instruction_label)
+
+        report = (
+            f"ilastik version: {ilastik_version}\n"
+            f"OS: {platform.platform()}\n"
+            f"{workflow_report_text}"
+            "Session log (please shorten to relevant parts if possible):\n"
+            f"```\n{_mask_file_paths(_get_current_session_log_or_hint())}\n```"
+        )
+        self.report_text = QTextEdit()
+        self.report_text.setPlainText(report)
+        self.report_text.setSizePolicy(QSizePolicy.MinimumExpanding, QSizePolicy.Minimum)
+        main_layout.addWidget(self.report_text)
+
+        log_path = ilastik_logging.get_logfile_path()
+        if log_path:
+            log_url = QUrl(pathlib.Path(log_path).parent.as_uri())
+            log_button = QPushButton("Open log folder")
+            log_button.clicked.connect(partial(QDesktopServices.openUrl, log_url))
+            main_layout.addWidget(log_button, alignment=Qt.AlignRight)
+
+        button_layout = QHBoxLayout()
+        copy_button = QPushButton("Copy and open forum")
+        copy_button.clicked.connect(self.copy_and_go_to_forum)
+        button_layout.addWidget(copy_button)
+        email_button = QPushButton("Open in email app")
+        email_button.clicked.connect(self.open_in_email_app)
+        button_layout.addWidget(email_button)
+        close_button = QPushButton("Close")
+        close_button.clicked.connect(self.close)
+        button_layout.addWidget(close_button)
+        main_layout.addLayout(button_layout)
+
+        # Set the layout to the dialog
+        self.setLayout(main_layout)
+        self.show()
+
+    def copy_and_go_to_forum(self):
+        clipboard = QApplication.clipboard()
+        clipboard.setText(self.report_text.toPlainText())
+        webbrowser.open(FORUM_URI)
+
+    def open_in_email_app(self):
+        subject = "[Bugreport] --Summarise your issue here--"
+        body = (
+            "Please describe the issue you are experiencing and adapt the email subject.\n"
+            "\n\n--\nAuto-generated report:\n"
+        )
+        body += self.report_text.toPlainText()
+        mailto_url = f"mailto:{TEAM_EMAIL}?subject={subject}&body={quote(body)}"
```

Comment:
```suggestion
        mailto_url = f"mailto:{TEAM_EMAIL}?subject={subject}&body={body}"
```

for me removing the `quote` call would open the message in the email app as expected. But I guess it's probably important to do in order to encode as a url....

Reply:
Can you check again? I suspect it actually wasn't too much, but rather not enough quoting.

Reply:
works now 😻